src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_help.py

```diff
@@ -261,3 +261,183 @@
         - name: --remove-action -r
           short-summary: Remove receivers from the action group. Accept space separated list of receiver names.
 """
+
+helps['monitor activity-log alert'] = """
+    type: group
+    short-summary: Manage activity log alerts
+"""
+
+helps['monitor activity-log alert list'] = """
+    type: command
+    short-summary: List activity log alerts under a resource group or the current subscription.
+    parameters:
+        - name: --resource-group -g
+          short-summary: Name of the resource group under which the activity log alerts are being listed. If it is
+                         omitted, all the activity log alerts under the current subscription are listed.
+"""
+
+helps['monitor activity-log alert create'] = """
+    type: command
+    short-summary: Create a default activity log alert
+    long-summary: This command will create a default activity log with one condition which compares if the activities
+                  logs 'category' field equals to 'ServiceHealth'. The newly created activity log alert does not have
+                  any action groups attached to it.
+    parameters:
+        - name: --name -n
+          short-summary: Name of the activity log alerts
+        - name: --scope -s
+          short-summary: A list of string that will be used as prefixes. The alert will only apply to activityLogs
+                         with resourceIds that fall under one of these prefixes. If not provided, the path to this
+                         resource group will be used.
+        - name: --disable
+          short-summary: Disable the activity log alert after it is created.
+        - name: --description
+          short-summary: A description of this activity log alert
+        - name: --condition -c
+          short-summary: A condition expression represents the condition that will cause the alert to activate. The
+                         format is FIELD=VALUE[ and FILED=VALUE...]. The possible values for the field are 'resourceId',
+                         'category', 'caller', 'level', 'operationName', 'resourceGroup', 'resourceProvider', 'status',
+                         'subStatus', 'resourceType', or anything beginning with 'properties.'.
+        - name: --action-group -a
+          short-summary: Add action group. Accept space separated action groups identifiers. The identify can be the
+                         action group's name (if it is in the same resource group as the alert) or its resource id.
```

Comment:
> (if it is in the same resource group as the alert)

This is the standard behavior of the CLI so you shouldn't need to call it out.

Reply:
Cool.